packages/contracts/contracts/plugin/examples/count/CounterV1PluginSetup.sol

```diff
@@ -71,15 +71,15 @@ contract CounterV1PluginSetup is PluginSetup {
             PermissionLib.Operation.Grant,
             _dao,
             plugin,
-            NO_ORACLE,
+            noOracle,
```

Comment:
if my previous comment is wrong and using an uninitialized variable is cheaper, then at least read it once into memory and then use it through the function